packages/suite/src/actions/suite/initAction.ts

```diff
@@ -66,13 +66,13 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     await dispatch(periodicCheckTokenDefinitionsThunk());
 
     // 8. init periodic fetching of fiat rates
-    await dispatch(
+    dispatch(
```

Comment:
Please remove it in another PR. This goes to current release and it is dangerous to do those changes now

Reply:
I am not sure how `periodicFetchFiatRatesThunk` will work with the new method you added

Reply:
Okey, actually even awaiting transaction fiat rates is not slowing down that much (tested on QA wallet)

Reply:
Done